### ISSUE TITLE
Accessibility: Page Headers <h1> not visible to screen reader

### ISSUE THREAD

**Issue by warpcoil** [brainstormforce/astra#5693]
Hi

Thank You for the provision of a fantastic product; never the less:

### Describe the bug:
Testing the page headers module of Astra Pro results in the first heading ```<h1>``` being missed by screen reader software. This is likely due to the fact the outer image has an ```aria-label=""```. The image is correctly described as "decorative" but the heading is now not visible to NVDA/Narrator.

### To reproduce:
Access any page where the Astra Pro Page Headers module is active (parallax/no parallax does matter).

```html
<div class="ast-title-bar-wrap ast-advanced-headers-vertical-center" role="img" aria-label="">
    <div class="ast-inside-advanced-header-content">
        <div class="ast-advanced-headers-layout ast-advanced-headers-layout-2">
            <div class="ast-container ast-title-bar-align-center">
                <div class="ast-advanced-headers-wrap">
                    <h1 class="ast-advanced-headers-title">

                        About Us

                    </h1>
                </div>
            </div>
        </div>
    </div>
</div>
```

### Expected behavior:
Expected: First and foremost heading ```<h1>``` to be readable by screen reader software. If I press H or Shift+H to navigate forwards and backwards through heading; I should expect "Animals" first, proceeding onto "Dogs" and breeds of dogs and then "Cats". No levels skipped or missing.

```html
<h1>Animals</h1>
    <h2>Dogs</h2>
        <h3>German Shepherd</h3>
        . . .
        <h3>Border Collie</h3>
    <h2>Cats</h2>
        <h3>Siamese</h3>
        <h3>Persian</h3>
```

### Screenshots:

![image](https://github.com/brainstormforce/astra/assets/5113027/0efbd152-0d0a-47e3-90ca-092697e8fb7f)

### Isolating the problem:

- [x] I can reproduce this bug consistently using the steps above

### WordPress version:

WordPress 6.4.2 (but applies to any)

### Astra Theme version:

Astra 4.6.2 (but applies to any)

### One possible workaround
I'm sure there's a better solution though (as I cannot edit the theme file).
```html
<div class="ast-title-bar-wrap ast-advanced-headers-vertical-center" role="img" aria-label="">
    <div class="ast-inside-advanced-header-content">
        <div class="ast-advanced-headers-layout ast-advanced-headers-layout-2">
            <div class="ast-container ast-title-bar-align-center">
                <div class="ast-advanced-headers-wrap">
                    <h1 class="ast-advanced-headers-title">

                        About Us

                    </h1>
                </div>
            </div>
        </div>
    </div>
</div>
<h1 class="screen-reader-text">About Us</h1>
```

### Why this works
* Only one h1 heading detected (as it should be)
* Screen reader software can "see" the text but we can't -- it's hidden
![image](https://github.com/brainstormforce/astra/assets/5113027/26487b5c-68c2-4832-8d24-f0ade50b7669)

### Why this doesn't work
* Duplicated data
* Possibly multiple called to ```php get_the_title()```;

I'll leave it to you to decide _how_ but we're legislatively obliged to conform to WCAG 2.1 (AA) as much as reasonably possible.

Thanks

Bobby

**Comment by yssambare12** [comment]
Hi @warpcoil 

Thank you for bringing this issue to our attention. We appreciate your patience and understanding. We are actively working on improving this case to ensure better accessibility for all users, including those who rely on screen readers. We will keep you updated on our progress and notify you once the improvement is implemented and available for use.

Thanks & regards

**Comment by warpcoil** [comment]
Thank You @yssambare12
